virttest/utils_disk.py

```diff
@@ -78,30 +79,32 @@ def is_mount(src, dst=None, fstype=None, options=None, verbose=False, session=No
     """
     mount_str = "%s %s %s" % (src, dst, fstype)
     mount_str = mount_str.replace("None", "").strip()
-    mount_list_cmd = "cat /proc/mounts"
+    mount_check_cmd = "findmnt -J %s" % mount_str
```

Comment:
I think if dst and fstype are specified, the current CMD does not work.

```bash
findmnt -h

Usage:
 findmnt [options]
 findmnt [options] <device> | <mountpoint>
 findmnt [options] <device> <mountpoint>
 findmnt [options] [--source <device>] [--target <path> | --mountpoint <dir>]
```

Reply:
Oh, I know what you mean, let me have a try

Reply:
Hi @PaulYuuu I updated the code, could you please review it again?Thanks.